pkg/sdk/secrets_def.go

```diff
@@ -241,15 +241,17 @@ var SecretsDef = g.NewInterface(
 		SQL("SECRETS").
 		OptionalLike().
 		OptionalExtendedIn(),
-).ShowByIdOperation().
-	DescribeOperation(
-		g.DescriptionMappingKindSingleValue,
-		"https://docs.snowflake.com/en/sql-reference/sql/desc-secret",
-		secretDetailsDbRow,
-		secretDetails,
-		g.NewQueryStruct("DescribeSecret").
-			Describe().
-			SQL("SECRET").
-			Name().
-			WithValidation(g.ValidIdentifier, "name"),
-	)
+).DescribeOperation(
+	g.DescriptionMappingKindSingleValue,
+	"https://docs.snowflake.com/en/sql-reference/sql/desc-secret",
+	secretDetailsDbRow,
+	secretDetails,
+	g.NewQueryStruct("DescribeSecret").
+		Describe().
+		SQL("SECRET").
+		Name().
+		WithValidation(g.ValidIdentifier, "name"),
+).ShowByIdOperation(
```

Comment:
nit: why the changed ordering? Now the Show methods are not next to each other in the generated interface.

Reply:
the order does not have to be changed, I will regenerate them with correct ordering